docs/machine-learning/resources/tasks.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Machine learning tasks
-description: Explore the different machine learning tasks supported in ML.NET.
+title: Machine learning tasks - ML.NET | Microsoft Docs
+description: Explore the different machine learning tasks and associated learners that are supported in ML.NET.
+ms.custom: seodec18
+ms.manager: cgronlun 
```

Comment:
this metadata field is not valid

https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master